electron/index.d.ts

```diff
@@ -1864,12 +1864,7 @@ declare namespace Electron {
 		/**
 		 * Writes data to the clipboard.
 		 */
-		write(data: {
-			text?: string;
-			rtf?: string;
-			html?: string;
-			image?: NativeImage;
-		}, type?: ClipboardType): void;
+		write(data: { text: string; bookmark?: string; } | { rtf: string; } | { html: string; } | { image: NativeImage; }, type?: ClipboardType): void;
```

Comment:
Should it be below? It looks that only `bookmark` is missing in current type definitions.

```typescript
write(data: {
		text?: string;
		rtf?: string;
		html?: string;
		image?: NativeImage;
		bookmark?: string;
	}, type?: ClipboardType): void;
```